config/slash_commands/esv.rb

```diff
@@ -0,0 +1,39 @@
+Houston::Slack.config do
+  slash("esv") do |e|
+    if e.text.blank?
+      # Print usage
+      e.respond! "No search query given. Usage: `/esv <reference>` where reference is a verse reference, e.g. John 3:16 (book abbreviations _are_ supported)."
```

Comment:
This is just my opinion, so _please_ feel free to take it or leave it. But since the rest of the company will likely use this, should the language be a little more humanized, like your lunch command or slackbot, and not as similar to a terminal error message. 

maybe `"Whoops! You didn't enter a Bible reference. ..."`

Or even include an example or two like slackbot
<img width="506" alt="slack" src="https://cloud.githubusercontent.com/assets/3615754/12683297/117b34ea-c67e-11e5-8a30-a24334f52355.png">